src/cookieconsent.js

```diff
@@ -1089,21 +990,18 @@
 
     // This runs the service located at index `currentServiceIndex`.
     // If the service fails, `runNextServiceOnError` will continue trying each service until all fail, or one completes successfully
-    Location.prototype.locate = function () {
+    Location.prototype.locate = function (resolve, reject) {
```

Comment:
they just resemble the names. `resolve` is a callback function that is called if the async is successful, and `reject` is called if the async code throws an error.

I could change them to `complete` and `error` or something?